src/sentry/models/integrations/integration.py

```diff
@@ -58,6 +58,7 @@ class Integration(DefaultFieldsModel):
     status = BoundedPositiveIntegerField(
         default=ObjectStatus.ACTIVE, choices=ObjectStatus.as_choices(), null=True
     )
+    date_updated = models.DateTimeField(auto_now=True, null=True)
```

Comment:
since this changes the type of this column (according to django) you can remove the inheritance from DefaultFieldsModel (this is what was done for the other conflicts on these date fields)